plugins/module_utils/network/sonic/config/ospfv3_interfaces/ospfv3_interfaces.py

```diff
@@ -0,0 +1,440 @@
+    {'config': {'name': '', '__delete_op': __DELETE_LEAFS_THEN_CONFIG_IF_NO_NON_KEY_LEAF}}
+]
+
+OSPF_INT_ATTRIBUTES = {
+    'bfd': {
+        'enable': '/enable-bfd',
+        'bfd_profile': '/enable-bfd/config/bfd-profile'
+    },
+    'network': '/config/network-type',
+    'area_id': '/config/area-id',
+    'cost': '/config/metric',
+    'dead_interval': '/config/dead-interval',
+    'hello_interval': '/config/hello-interval',
+    'mtu_ignore': '/config/mtu-ignore',
+    'priority': '/config/priority',
+    'retransmit_interval': '/config/retransmission-interval',
+    'transmit_delay': '/config/transmit-delay',
+    'passive': '/config/passive',
+    'advertise': '/config/advertise'
+
+}
+
+
+class Ospfv3_interfaces(ConfigBase):
+    """
+    The sonic_ospfv3_interfaces class
+    """
+
+    gather_subset = [
+        '!all',
+        '!min',
+    ]
+
+    gather_network_resources = [
+        'ospfv3_interfaces',
+    ]
+
+    def __init__(self, module):
+        super(Ospfv3_interfaces, self).__init__(module)
+
+    def get_ospfv3_interfaces_facts(self):
+        """ Get the 'facts' (the current configuration)
+        :rtype: A dictionary
+        :returns: The current configuration as a dictionary
+        """
+        facts, _warnings = Facts(self._module).get_facts(self.gather_subset, self.gather_network_resources)
+        ospfv3_interfaces_facts = facts['ansible_network_resources'].get('ospfv3_interfaces')
+        if not ospfv3_interfaces_facts:
+            return []
+        return ospfv3_interfaces_facts
+
+    def execute_module(self):
+        """ Execute the module
+        :rtype: A dictionary
+        :returns: The result from module execution
+        """
+        result = {'changed': False}
+        warnings = list()
+        commands = list()
+        latest_config = []
+
+        existing_ospfv3_interfaces_facts = self.get_ospfv3_interfaces_facts()
+        commands, requests = self.set_config(existing_ospfv3_interfaces_facts)
+        if commands and len(requests) > 0:
+            if not self._module.check_mode:
+                try:
+                    edit_config(self._module, to_request(self._module, requests))
+                except ConnectionError as exc:
+                    self._module.fail_json(msg=str(exc), code=exc.code)
+            result['changed'] = True
+        result['commands'] = commands
+
+        changed_ospfv3_interfaces_facts = self.get_ospfv3_interfaces_facts()
+
+        result['before'] = existing_ospfv3_interfaces_facts
+        if result['changed']:
+            result['after'] = changed_ospfv3_interfaces_facts
+
+        new_config = changed_ospfv3_interfaces_facts
+        old_config = existing_ospfv3_interfaces_facts
+        if self._module.check_mode:
+            result.pop('after', None)
+            new_commands = deepcopy(commands)
+
+            new_config = get_new_config(new_commands, old_config, TEST_KEYS)
+
+            latest_config = self.new_cfg(new_config)
```

Comment:
```suggestion
            new_config = self.new_cfg(new_config)
```

Reply:
replace 'latest_config' in below follow ups as well.

Reply:
This change is not resolved.
You changed latest_config to new_config in this, but didnt do the same in the followup